apps/admin-api/env/.env.sample

```
@@ -0,0 +1,44 @@
+# ============================================================================
+# Database Configuration
+# ============================================================================
+POSTGRES_DB_URI=postgresql://postgres:password@localhost:5432/console-users
+
+# ============================================================================
+# Server Configuration
+# ============================================================================
+PORT=3010
+CORS_WEBSITE_URLS=http://localhost:3011
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORS_WEBSITE_URLS key should go before the PORT key

(UnorderedKey)